nbgrader/apps/api.py

```diff
@@ -861,6 +861,36 @@ def assign(self, assignment_id, force=True, create=True):
             app.create_assignment = create
             return capture_log(app)
 
+    def feedback(self, assignment_id, force=True, create=True):
+        """Run ``nbgrader feedback`` for a particular assignment.
+
+        Arguments
+        ---------
+        assignment_id: string
+            The name of the assignment
+        force: bool
+            Whether to force creating the student version, even if it already
+            exists.
+        create: bool
+            Whether to create the assignment in the database, if it doesn't
+            already exist.
+
+        Returns
+        -------
+        result: dict
+            A dictionary with the following keys (error and log may or may not be present):
+
+            - success (bool): whether or not the operation completed successfully
+            - error (string): formatted traceback
+            - log (string): captured log output
+
+        """
+        with temp_attrs(self.coursedir, assignment_id=assignment_id):
+            app = Feedback(coursedir=self.coursedir, parent=self)
+            app.force = force
+            app.create_assignment = create
```

Comment:
I think `create_assignment` is only relevant for `nbgrader assign` and won't do anything for `nbgrader feedback`, so you should be able to remove this line.